configs/gateway/nginx-http.conf

```diff
@@ -4,6 +4,15 @@ server {
 
     server_name _;
 
+    # enable gzip
+    gzip on;
+    gzip_buffers 16 8k;
```

Comment:
I'm not convinced this is large enough. `common.js` is the largest file that we serve and 16 8K buffers is only 131K in size which isn't big enough? That being said, I cannot replicate an issue serving the file under the current config... 🤔 

I'm also wondering whether there is any potential issue with an 8K buffer size. nginx docs say it uses 4K or 8K depending on memory page size. The default is set to the below in v0.7.28+:
```
the default "gzip_buffers" directive values have been changed to 32 4k or 16 8k from 4 4k/8k.
```

http://nginx.org/en/docs/http/ngx_http_gzip_module.html

Reply:
https://forum.nginx.org/read.php?2,239316,239344#msg-239344 seems to suggest it won't truncate

Reply:
I think we just use the default set by nginx (32 4K / 16 8K depending on platform) rather than hardcode 16 8K 🤷